CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+### 0.12.9
+*Features*
+- Added a helper function to allow retrying blocks of code
+
+*Bug Fixes*
+- Retry discovering namespace and kubernetes context
+- Expose real error during namespace discovery
+
+### 0.12.8
+*Features*
```

Comment:
This was actually a bugfix--Make Deployment use its own hard timeout instead of ReplicaSet's by simplifying timeout logic